electrum/commands.py

```diff
@@ -352,6 +352,17 @@ async def setconfig(self, key, value):
             cv = self.config.cv.from_key(key)
             cv.set(value)
 
+    @command('')
+    async def listconfig(self):
+        """Returns the list of all configuration variables. """
+        return self.config.list_config_vars()
+
+    @command('')
+    async def helpconfig(self, key, more=False):
+        """Returns help about a configuration variable. """
+        cv = self.config.cv.from_key(key)
+        return cv.get_long_desc() if more else cv.get_short_desc()
```

Comment:
I am not sure we need the `--more` option. How about combining the two descriptions?
e.g.:
```suggestion
    async def helpconfig(self, key):
        """Returns help about a configuration variable. """
        cv = self.config.cv.from_key(key)
        short = cv.get_short_desc()
        long = cv.get_long_desc()
        if short and long:
            return short + "\n---\n\n" + long
        else:
            return short or long
```

Reply:
indeed, that's better